src/arch/xtensa/include/arch/sof.h

```diff
@@ -52,58 +55,33 @@ static inline void *arch_get_stack_ptr(void)
 	return ptr;
 }
 
+static inline void fill_core_dump(struct sof_ipc_dsp_oops_xtensa *oops,
+				   uint32_t ps)
+{
+	oops->hdr.arch = 0; // TODO: Put specific values for different archs
```

Comment:
@lgirdwood
Does this align with what you had on mind?
How should I assign values to different architectures and where to define them?

Reply:
@lgirdwood Should I wait for your PR to be merged and rebase on top of it, or redirect my PR on topic/abi, so that these changes would get merged together?
I would also appreciate a proposal for architecture encoding scheme, to bring implementation of feature to finish.